roles/analytics-dashboard/tasks/main.yml

```diff
@@ -58,17 +69,32 @@
         remote_src: yes
       when: pack_location == "remote"
 
-#    - name: Copy JDBC Driver
-#      copy:
-#       src: "{{ product_package_location }}/lib/{{ jdbc_driver }}"
-#       dest: "{{ carbon_home }}/repository/components/lib"
+    - name: Copy JDBC Driver
+      copy:
+       src: "{{ product_package_location }}/lib/{{ jdbc_driver }}"
+       dest: "{{ carbon_home }}/wso2/analytics/lib"
```

Comment:
Check if JDBC driver is defined

Reply:
fixed https://github.com/wso2/ansible-ei/pull/30/commits/bf084ed7e012838529727ccbb5adb33110a47243